### PR TITLE
Fixed the build

### PR DESCRIPTION
**Pull request by zr3**
Updated packages, fixed some breaking changes, added some linter rules for ubiquitous less issues and fixed some easy issues.

This addresses issue #1837

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,25 +4,25 @@
...
-    "rollup-plugin-node-resolve": "*",
-    "rollup-plugin-nodent": "*",
-    "uglify-js": "*",
-    "preprocess-cli-tool": "*",
-    "less": "^2.0.0",
-    "less-plugin-autoprefix": "*",
-    "stylelint": "^8.2.0",
-    "stylelint-config-recommended": "^1.0.0",
-    "svgo": "*"
+    "babel-eslint": "^10.0.3",
```
Comment (zr3) [inline package.json]: These are all the current latest versions

File: src/config/eslint.config.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
+	"parser": "babel-eslint",
```
Comment (zr3) [inline src/config/eslint.config.json]: https://stackoverflow.com/questions/36001552/eslint-parsing-error-unexpected-token is the issue that this addresses

**Comment by mikekasprzak** [comment]
Okay, I'll need a block of time to test these changes on the live server. 99% sure I'll have to update the packages. I'll try to do it in the next couple days.

**Comment by zr3** [comment]
Sounds good to me -- and yeah, I agree that it would definitely be good to merge this when there's not a lot of traffic to the site, if possible. FWIW I was able to spot-check the home page and the main tabs to verify that it was working and that the js and css compiled correctly, but I haven't set up the database yet, so I didn't go any deeper than that.

I also didn't look at any of the sites that aren't ldjam.com/ludumdare.org if they're relevant, still working on familiarizing myself with the project 😊

**Comment by local-minimum** [comment]
@zr3 there's a helper script that will simulate an event which should set up the databases.

But also @mikekasprzak, isn't deployment manual? In that case couldn't we merge it into master without causing any issues and at the same time making it easier for me people to poke around the code?

**Comment by nicolasbize** [comment]
Just checked out the code for the first time and ran into the build issue straight away.
Setting a correct parser for ESLint did the trick indeed.

I'm curious, how do you guys do deployments / prod db changes?

**Pull request merged**